components/navbar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Logo from './../public/mcssLogo.svg'
+import Image from 'next/image'
+
+export const NavBar = (): React.ReactElement => {
+    return (
+        <>
+            <nav className={"flex items-center justify-between flex-wrap bg-teal-500 mt-14 ml-24 mr-24"}>
+                <div className={"flex item-center flex-shrink-0 mr-6"}>
+                    <Image src={Logo} alt={"MCSS logo"}></Image>
+                </div>
+                <div className={"w-full block flex-grow lg:flex lg:items-center lg:w-auto"}>
```

Comment:
```suggestion
                <div className="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
```
This also appears in a few other places

Reply:
Fixed, thanks